services/capture/Dockerfile

```diff
@@ -7,7 +7,7 @@ FROM openjdk:17.0.1-slim AS final
 WORKDIR /app
 COPY --from=base /project/target .
 
-RUN apt-get update && apt-get install curl -y
+RUN apt-get update && apt-get install curl -y && apt-get install procps -y
```

Comment:
I think you can do this in less steps.
`RUN apt-get update && apt-get install curl procps -y`

Reply:
Fixed it.